### PR TITLE
Get album

### PR DESCRIPTION
**Pull request by FrancoCoronel**
## Summary

- Get album information by id
- Add album query
- Add album type

## Trello Card
https://trello.com/c/d5NA50Ev/13-traer-un-album

### CODE REVIEW

File: app/graphql/albums/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { gql } = require('apollo-server');
+
+module.exports = {
```
Comment (igna92ts) [inline app/graphql/albums/index.js]: why is this here?

File: app/services/albums.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  photosService = require('./photos');
+
+const { ALBUM_API_URL } = constants;
+
+exports.getAlbum = async id => {
+  const url = `${ALBUM_API_URL}/albums?id=${id}`;
+  logger.info(`Making a request to url ${url}`);
+  try {
+    const albumInfo = await axios.get(`${url}`);
+    if (!albumInfo.data.lenght) {
```
Comment (igna92ts) [inline app/services/albums.js]: typo right here, remember to always test all cases by hand on top of your automatic tests

File: app/services/albums.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      throw errors.notFound(`Album with id ${id} not found`);
+    }
+    const photosForAlbum = await photosService.getPhotosForAlbum(id);
+    return {
+      id: albumInfo.data[0].id,
```
Comment (igna92ts) [inline app/services/albums.js]: save `albumInfo.data[0]` in a variable with a shorter name to make code more concise and to avoid indexing many times

File: config/index.js [hunk 4]
```diff
@@ -46,6 +46,13 @@ const config = {
     session: {
       header_name: 'authorization',
       secret: process.env.NODE_API_SESSION_SECRET
+    },
+    albumApi: {
+      url: process.env.ALBUM_API_URL,
+      routes: {
+        albums: process.env.ALBUMS_URL,
```
Comment (igna92ts) [inline config/index.js]: you dont seem to be using this 2

**Review by igna92ts — CHANGES_REQUESTED** [review]

File: app/graphql/albums/index.js (hunk 1 above)
Comment (FrancoCoronel) [reply app/graphql/albums/index.js]: My mistake!

**Review by igna92ts — APPROVED** [review]
